program/src/processor.rs

```diff
@@ -1652,6 +1660,10 @@ fn checked_add_borrow(
 ) -> MangoResult<()> {
     margin_account.checked_add_borrow(token_index, quantity)?;
     mango_group.checked_add_borrow(token_index, quantity)
+
```

Comment:
What if there's an error  on mango_group.checked_add_borrow? I don't think it will return it